step2-07/demo/src/components/TodoList.tsx

```diff
@@ -21,17 +21,12 @@ const TodoList = (props: TodoListProps) => {
   );
 };
 
-function mapStateToProps(state: Store) {
-  return { ...state };
-}
-
-function mapDispatchToProps(dispatch: any) {
```

Comment:
Removed mapDispatchToProps here since it's unused (also demonstrates that the prop is optional)